zerver/webhooks/bitbucket3/view.py

```diff
@@ -114,25 +138,153 @@ def repo_push_handler(payload: Dict[str, Any], branches: Optional[str]=None) ->
+                                          title=pr["title"] if include_title else None)
+
+def get_pr_opened_or_modified_body(payload: Dict[str, Any], action: str,
+                                   include_title: Optional[bool]) -> str:
+    pr = payload["pullRequest"]
+    description = pr.get("description")
+    assignees_string = get_assignees_string(pr)
+    if assignees_string:
+        # Then use the custom message template for this particular integration so that we can
+        # specify the reviewers at the end of the message (but before the description/message).
+        parameters = {"user_name": get_user_name(payload),
+                      "action": action,
+                      "url": pr["links"]["self"][0]["href"],
+                      "number": pr["id"],
+                      "source": pr["fromRef"]["displayId"],
+                      "destination": pr["toRef"]["displayId"],
+                      "message": description,
+                      "assignees": assignees_string,
+                      "title": pr["title"] if include_title else None}
+        if include_title:
+            body = PULL_REQUEST_OPENED_TEMPLATE_WITH_REVIEWERS_WITH_TITLE.format(**parameters)
+        else:
+            body = PULL_REQUEST_OPENED_TEMPLATE_WITH_REVIEWERS.format(**parameters)
+        if description:
+            body += '\n' + CONTENT_MESSAGE_TEMPLATE.format(message=description)
+        return body
+    return get_pull_request_event_message(user_name=get_user_name(payload),
+                                          action=action,
+                                          url=pr["links"]["self"][0]["href"],
+                                          number=pr["id"],
+                                          target_branch=pr["fromRef"]["displayId"],
+                                          base_branch=pr["toRef"]["displayId"],
+                                          message=pr.get("description"),
+                                          assignee=assignees_string if assignees_string else None,
+                                          title=pr["title"] if include_title else None)
+
+def get_pr_needs_work_body(payload: Dict[str, Any], include_title: Optional[bool]) -> str:
+    pr = payload["pullRequest"]
+    if not include_title:
+        return PULL_REQUEST_MARKED_AS_NEEDS_WORK_TEMPLATE.format(user_name=get_user_name(payload),
+                                                                 number=pr["id"],
+                                                                 url=pr["links"]["self"][0]["href"])
+    return PULL_REQUEST_MARKED_AS_NEEDS_WORK_TEMPLATE_WITH_TITLE.format(user_name=get_user_name(payload),
+                                                                        number=pr["id"],
+                                                                        url=pr["links"]["self"][0]["href"],
+                                                                        title=pr["title"])
+
```

Comment:
As an example, this last `return` statement is unnecessarily long and would be a pain to read if I was working on two files side by side! :)

Reply:
> Note that it is already a pain to read because even GitHub had to render it with a horizontal scrollbar. :)

Definitely :sweat_smile: